### DISCUSSION TITLE
admin密码忘记了，如何重置？

### DISCUSSION THREAD

**Discussion by wuzhi-dev** [flagify-com/OctoMation#60] [General]
admin密码忘记了，如何重置？

**Comment by wzfukui** [comment]
你好，请参考[运维手册-重置admin密码](https://github.com/flagify-com/OctoMation/wiki/OctoMation%E8%BF%90%E7%BB%B4%E6%89%8B%E5%86%8C#%E9%87%8D%E7%BD%AEadmin%E5%AF%86%E7%A0%81)重置`admin`用户的密码为默认的`octomation`。

**注意：重置密码后，务必将默认密码修改成一个健壮的新密码！**
